executor/executor_test.go

```diff
@@ -3801,6 +3801,32 @@ func (s *testSuite) TestSplitIndexRegion(c *C) {
 	c.Assert(err, NotNil)
 	terr := errors.Cause(err).(*terror.Error)
 	c.Assert(terr.Code(), Equals, terror.ErrCode(mysql.WarnDataTruncated))
+
+	// Check min value is more than max value.
+	tk.MustExec(`split table t index idx1 between (0) and (1000000000) regions 10`)
```

Comment:
```
tidb> create table t(a int, b int, key b_idx(b));
Query OK, 0 rows affected (0.00 sec)

tidb> split table t index b_idx between (aa) and (ac) regions 10;
ERROR 1054 (42S22): Unknown column 'aa' in ''

tidb> split table t index b_idx between ('ac') and (ac) regions 10;
ERROR 1265 (01000): Data Truncated
```
Can we refine the error message?

Reply:
Great catch! Thanks.

I think the error of `Unknown column` is reasonable.
```sql
root:test> create table t (a int, b int,index idx(a));
Query OK, 0 rows affected
Time: 0.065s
root:test> insert into t values (aa,bb);
(1054, u"Unknown column 'aa' in ''")
root:test>
```
I will try to refine the error of `Data Truncated`.
@bb7133